cmd/erasure-multipart.go

```diff
@@ -309,20 +309,36 @@ func (er erasureObjects) ListMultipartUploads(ctx context.Context, bucket, objec
 		if populatedUploadIds.Contains(uploadID) {
 			continue
 		}
-		fi, err := disk.ReadVersion(ctx, minioMetaMultipartBucket, pathJoin(er.getUploadIDDir(bucket, object, uploadID)), "", false)
-		if err != nil {
-			if !IsErrIgnored(err, errFileNotFound, errDiskNotFound) {
-				logger.LogIf(ctx, err)
+		// If present, use time stored in ID.
+		if split := strings.Split(uploadID, "."); len(split) == 2 {
+			startTime, err := strconv.ParseInt(split[1], 10, 64)
+			if err != nil {
+				continue
 			}
-			// Ignore this invalid upload-id since we are listing here
-			continue
+			uploads = append(uploads, MultipartInfo{
+				Bucket:    bucket,
+				Object:    object,
+				UploadID:  base64.RawURLEncoding.EncodeToString([]byte(fmt.Sprintf("%s.%s", globalDeploymentID(), uploadID))),
+				Initiated: time.Unix(0, startTime),
+			})
+		} else {
+			// No new parts should be created without time.
```

Comment:
We don't need to care for existing ones @klauspost we can simply ignore and move on. 

Less code to carry.